includes/class-omise-capabilities.php

```diff
@@ -61,6 +61,18 @@ public function getBackends( $currency = '' ) {
 		return $this->capabilities->getBackends( $params );
 	}
 
+	/**
+	 * Retrieves details of Touch n Go payment backends from capabilities.
+	 *
+	 * @return string
+	 */
+	public function getTouchNGoBackends() {
```

Comment:
do we really need this one as i say is will be 
```json
            "touch_n_go": {
                "type": "touch_n_go",
                "currencies": [
                    "jpy",
                    "sgd",
                    "myr",
                    "thb"
                ],
                "amount": {
                    "min": 3000,
                    "max": 1000000
                },
                "provider": "RMS"
            }
```

Reply:
I use it for switching the title here https://github.com/omise/omise-woocommerce/blob/b3e20886d48e3f77e411c6e477865dcaaff47a0c/includes/backends/class-omise-backend-touch-n-go.php#L21